localisation/00_wc_wonders_upgrades.csv

```diff
@@ -48,6 +48,32 @@ upgrade_silithid_greathive;Great Silithid Hive;Grande ruche silithide;Gro
 upgrade_silithid_greathive_desc;The great silithid hive located in Ahn'Qiraj houses the biggest of their queens who is constantly laying eggs to produce armies of inscetoid warriors.;La grande ruche silithide d'Ahn'Qiraj abrite la plus grande de leurs reines, qui pond constamment des �ufs pour produire des arm�es de guerriers insc�to�des.;Der gro�e Silithiden-Schwarmstock in Ahn'Qiraj beherbergt die gr��te ihrer K�niginnen, die unerm�dlich Eier legt, um Armeen insektoider Krieger zu produzieren.;;;;;;;;;;;x
 upgrade_crucible_of_storms;Crucible of Storms;Creuset des Temp�tes;Tiegel der St�rme;;;;;;;;;;;x
 upgrade_crucible_of_storms_desc;A long-submerged evil stirs beneath the Shrine of the Storm. The whispers that drive to madness echo within the depths, a siren song of eldritch power. Those who listen and obey prepare dark rituals to carry out their master�s will.;Un mal longtemps englouti s'agite sous le Sanctuaire des Temp�tes. Les murmures qui poussent � la folie r�sonnent dans les profondeurs, un chant de sir�ne de la puissance des anciens. Ceux qui �coutent et ob�issent pr�parent de sombres rituels pour ex�cuter la volont� de leur ma�tre.;Unter dem Heiligtum des Sturms r�hrt sich ein lange versunkenes �bel. In den Tiefen hallt das in den Wahnsinn treibende Fl�stern wider, ein Sirenengesang von �berirdischer Kraft. Diejenigen, die lauschen und gehorchen, bereiten dunkle Rituale vor, um den Willen ihres Meisters zu erf�llen.;;;;;;;;;;;x
+upgrade_queens_court;Queen's Court;;;;;;;;;;;;;x
+upgrade_queens_court_desc;For ten thousand years, the Queen's Court has never failed to enforce the word of Azshara.;;;;;;;;;;;;;x
+upgrade_darkest_depths;Darkest Depths;;;;;;;;;;;;;x
+upgrade_darkest_depths_desc;An abyss so dark that not even the naga dare to venture into it.;;;;;;;;;;;;;x
+upgrade_azshara_bedchamber;Azshara's Bedchamber;;;;;;;;;;;;;x
+upgrade_azshara_bedchamber_desc;The sleeping chamber of Queen Azshara. No being that has tried peeking inside lived to tell the tale.;;;;;;;;;;;;;x
+upgrade_hatchery;Hatchery;;;;;;;;;;;;;x
+upgrade_hatchery_desc;Both Naga and their monstrous pets are incubated here.;;;;;;;;;;;;;x
+upgrade_queen_statue;Queen's Statue;;;;;;;;;;;;;x
+upgrade_queens_statue_desc;A giant statue of The Light Beneath the Tides.;;;;;;;;;;;;;x
+upgrade_royal_guard_barracks;Royal Guard Barracks;;;;;;;;;;;;;x
+upgrade_royal_guard_barracks_desc;The quarters of Royal Guard of Her Radiance.;;;;;;;;;;;;;x
+upgrade_gate_of_the_queen;Gate of the Queen;;;;;;;;;;;;;x
+upgrade_gate_of_the_queen_desc;All who enter the magnificent Eternal Palace have to pass through these gates.;;;;;;;;;;;;;x
+upgrade_royal_treasury;Royal Treasury;;;;;;;;;;;;;x
+upgrade_royal_treasury_desc;The greatest treasures and wonders of the seven seas are stored here.;;;;;;;;;;;;;x
```

Comment:
Are there 7 seas on Azeroth?

Reply:
depending on your definition of a sea, there are five to eleven. in order:
Forbidding
Great
--Baradin Bay
--Coral
--Frozen
--South
----Mistveil
----Westersea
North
Uncharted
Veiled

Everything dashed is a 'sub'-sea inside another

i think five seas is the best option since its the option with the largest bodies of water

Reply:
we dont really have seven seas on earth either and its just a way of counting them to fit the popular phrase